phoenicis-tools/src/main/java/org/phoenicis/tools/http/Downloader.java

```diff
@@ -256,6 +256,7 @@ private void get(URL url,
                 connection.setHeaders(headers);
             }
 
+            connection.connect();
```

Comment:
Looks like you should merge master into the branch.

Reply:
Solved